src/styles/components/card/_card-body.scss

```diff
@@ -53,6 +53,7 @@
     h4,
     h5,
     h6 {
+      word-wrap: break-word;
```

Comment:
For any kind of situations, we're avoiding breaking words from the middle. This is a solution for the texts without spaces however it will also cause normal texts to break in the middle of the words. To avoid this, can we please apply the below solution? (including the line-height checks)

```css
word-break: normal;
overflow-wrap: break-word;
line-height: calc(var(--mynah-line-height) * 1.25);
```

Reply:
Resolved, thank you!